### PR TITLE
Refactor(handler part 9):  service generate

### PR DESCRIPTION
**Pull request by 1larity**
## Summary
This PR continues handler functional decomposition (Discussion #408) by extracting `service_generate` orchestration out of `acestep/handler.py` into focused mixins while preserving behavior and public interfaces.

### Extracted modules
- `acestep/core/generation/handler/service_generate_request.py`
  - input normalization
  - turbo step clamp
  - seed-list normalization/padding/truncation
- `acestep/core/generation/handler/service_generate_execute.py`
  - preprocess payload unpack
  - generation kwargs construction
  - MLX/PyTorch diffusion execution + fallback
- `acestep/core/generation/handler/service_generate_outputs.py`
  - output payload attachment for downstream consumers

### Wiring updates
- `acestep/handler.py`
  - `service_generate` converted to a thin orchestrator calling extracted helpers
  - expanded `service_generate` docstring (Args/Returns)
- `acestep/core/generation/handler/__init__.py`
  - exports for new mixins

## Review Fixes Applied
Addressed review findings and follow-ups in this branch:
1. Removed unintended behavior change:
   - stopped forwarding `disable_tqdm` in MLX diffusion call to preserve prior behavior parity.
2. Added missing edge-case tests:
   - seed list padding when shorter than batch
   - seed list truncation when longer than batch
   - lyrics truncation when longer than captions
   - `_resolve_service_seed_param(None)` random-seed path
3. Improved logging format correctness:
   - fixed Loguru format from `%s` to `{}` in PyTorch diffusion log line.
4. Maintained documentation standard:
   - staged Python symbols have docstring coverage.

### Automated
- Syntax checks:
  - `python -m py_compile acestep/core/generation/handler/service_generate_request.py`
  - `python -m py_compile acestep/core/generation/handler/service_generate_execute.py`
  - `python -m py_compile acestep/core/generation/handler/service_generate_outputs.py`
  - `python -m py_compile acestep/core/generation/handler/service_generate_request_test.py`
  - `python -m py_compile acestep/core/generation/handler/service_generate_execute_test.py`
  - `python -m py_compile acestep/handler.py`
  - `python -m py_compile acestep/core/generation/handler/__init__.py`
- Docstring audit on staged Python symbols: **0 missing**.

Added tests in this PR
test_normalize_inputs_clamps_turbo_steps_and_expands_lists
test_build_service_seed_list_duplicates_scalar_seed
test_seed_list_shorter_than_batch_gets_padded
test_seed_list_longer_than_batch_gets_truncated
test_lyrics_longer_than_batch_get_truncated
test_build_generate_kwargs_adds_timesteps_tensor
test_attach_service_outputs_persists_required_fields
test_resolve_seed_param_none_uses_random_seed

### Manual
Manual UI tests were run and passed:
- `Custom` generation flow
- `Remix` generation flow
- `Repaint` generation flow
- output rendering / result path checks (including auto-score/auto-lrc usage in normal UI flow)

## Scope / Risk
- Scope is limited to `service_generate` decomposition and related tests.
- Public interfaces are preserved.
- Non-target paths were not intentionally changed.
- Low risk; mostly structural extraction with parity checks and targeted tests.

## Summary by CodeRabbit

* **Refactor**
  * Restructured the service generation workflow to improve modularity and code organization by introducing specialized handler components.
  * Enhanced input normalization with improved turbo mode detection and seed parameter handling.
  * Optimized intermediate tensor assembly and output management for generation pipelines.

* **Tests**
  * Added comprehensive unit tests for service generation request normalization, execution, and output handling.

### CODE REVIEW

File: acestep/handler.py [hunk 1]
```diff
@@ -962,279 +968,101 @@ def service_generate(
...
+            lyrics=normalized["lyrics"],
+            keys=normalized["keys"],
+            target_wavs=target_wavs,
+            refer_audios=refer_audios,
+            metas=normalized["metas"],
+            vocal_languages=normalized["vocal_languages"],
+            repainting_start=normalized["repainting_start"],
+            repainting_end=normalized["repainting_end"],
+            instructions=normalized["instructions"],
+            audio_code_hints=normalized["audio_code_hints"],
```
[the remark this answers is not in the document]
Comment (1larity) [reply acestep/handler.py:1027]: Pre-existing point is valid as a robustness improvement, but it is not introduced by this decomposition.

[the remark this answers is not in the document]
Comment (1larity) [reply acestep/handler.py:1027]: Please raise an issue for this.

**Review by ChuxiJ — APPROVED** [review]
